servers/labscript/src/labscript_server/main.py

```
@@ -0,0 +1,136 @@
+                     with script_path).
+        script_path: Path to a .py labscript file on disk (mutually exclusive with
+                     script).
+        globals:     Optional flat dict of parameter names -> values to inject into
+                     the script's global namespace (e.g. {"duration": 1.0,
+                     "frequency": 2.5e6}).
+
+    Returns:
+        A string containing exit code, stdout, and stderr.
+    """
+    if script is not None and script_path is not None:
+        raise ValueError("Provide either 'script' or 'script_path', not both.")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)